cea/technologies/substation.py

```diff
@@ -7,9 +7,9 @@
 import numpy as np
 import scipy
 
-__author__ = "Jimeno A. Fonseca"
-__copyright__ = "Copyright 2015, Architecture and Building Systems - ETH Zurich"
-__credits__ = ["Jimeno A. Fonseca", "Tim Vollrath", "Thuy-An Nguyen"]
+__author__ = "Sreepathi Bhargava Krishna"
```

Comment:
Are you sure you're the author of this file?? Please ask the original author for permission to change authorship.